src/content/docs/apm/agents/nodejs-agent/installation-configuration/nodejs-agent-configuration.mdx

```diff
@@ -919,6 +919,219 @@ This section defines the Node.js agent variables in the order they typically app
   </Collapser>
 </CollapserGroup>
 
+## AI monitoring [#ai-monitoring]
+
+This section includes Node.js agent configurations for setting up AI monitoring. 
+
+<Callout variant="important">
+You need to set [distributed_tracing](/docs/apm/agents/nodejs-agent/installation-configuration/nodejs-agent-configuration/#dt-main) to `true` to capture trace and feedback data. It is turned on by default in Node.js agents 8.3.0 and higher. 
+</Callout>
+
+<CollapserGroup>
+    <Collapser
+        id="ai-monitoring-enabled"
+        title="ai_monitoring.enabled"
+    >
+    <table>
+      <tbody>
+        <tr>
+          <th>
+            Type
+          </th>
+
+          <td>
+            Boolean
+          </td>
+        </tr>
+
+        <tr>
+          <th>
+            Default
+          </th>
+
+          <td>
+            `false`
+          </td>
+        </tr>
+
+        <tr>
+          <th>
+            [Environ variable](#environment)
+          </th>
+
+          <td>
+            `NEW_RELIC_AI_MONITORING_ENABLED`
+          </td>
+        </tr>
+      </tbody>
+    </table>
+
+    When set to `true`, enables AI monitoring.
+        </Collapser>
+    <Collapser
```

Comment:
this is already documented, this can be removed. If you want to call out specifying a different value that's fine but this level of detail is redundant